cpu/cc2538/include/periph_cpu.h

```diff
@@ -73,6 +73,31 @@ typedef struct {
     gpio_t cs_pin;          /**< pin used for CS */
 } periph_spi_conf_t;
 
+/**
+ * @brief   Override resolution options
+ */
+#define HAVE_ADC_RES_T
+typedef enum {
+    ADC_RES_6BIT  = 0xa00,          /**< not supported by hardware */
+    ADC_RES_7BIT  = (0 << 4),       /**< ADC resolution: 7 bit */
+    ADC_RES_8BIT  = 0xb00,          /**< not supported by hardware */
+    ADC_RES_9BIT  = (1 << 4),       /**< ADC resolution: 9 bit */
+    ADC_RES_10BIT = (2 << 4),       /**< ADC resolution: 10 bit */
```

Comment:
Using `tests/periph_adc` the ADCs are initialized with a sampling width of 10 bit. However, applying 3,3 V to `ADC_2` runs into [this case](https://github.com/RIOT-OS/RIOT/blob/master/tests/periph_adc/main.c#L55). I didn't check if this is in cause of the normalisation to mV or the actual sample.